task.js

```diff
@@ -72,7 +72,39 @@ module.exports = {
     saveCoverage(coverageMap)
     debug('wrote coverage file %s', nycFilename)
 
-    return null
+    // count current code coverage
+    const counters = {
+      s: 0,
```

Comment:
Instead of `s, f, b`, could `statements, functions, branches` be used instead? The short names are kinda obscure

Reply:
good point, I will keep the one letter properties, since they match the coverage object properties, but will add comments